src/app/project/cac/become-a-member.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/toPromise';
+import 'rxjs/add/observable/forkJoin';
+
+import { ProjectService } from 'app/services/project.service';
+import * as moment from 'moment-timezone';
+import { Project } from 'app/models/project';
+import { ConfigService } from 'app/services/config.service';
+import { AbstractControl } from '@angular/forms';
+
+@Component({
+  templateUrl: './become-a-member.component.html',
+  styleUrls: ['./become-a-member.component.scss']
+})
+
+export class BecomeAMemberComponent implements OnInit {
+  @Input() project: Project;
+
+  public submitting = false;
+  private progressValue: number;
+  private progressBufferValue: number;
+  public totalSize: number;
+  public currentPage = 1;
+  private comment: Comment;
+  public acknowledged: any;
+
+  // CAC
+  public nameInput: string;
+  public emailInput: any;
+  public emailConfirmInput: any;
+  public liveNear: boolean;
+  public liveNearInput: any;
+  public memberOf: boolean;
+  public memberOfInput: any;
+  public knowledgeOf: boolean;
+  public knowledgeOfInput: any;
+  public additionalNotesInput: any;
+
+  constructor(
+    public activeModal: NgbActiveModal,
+    private projectService: ProjectService,
+  ) { }
+
+  ngOnInit() {
+    this.comment = new Comment();
+  }
+
+  private p1_next() {
+    this.currentPage++;
+  }
+
+  // TODO: Have some null checks in here from the front end
```

Comment:
Do null checks still need to be added or is this TODO complete?

Reply:
Removing comment, fixed in 974030a.